DFC.App.ExploreCareers.Repository.CosmosDb/CosmosDbConnection.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace DFC.App.ExploreCareers.Repository.CosmosDb
+{
+    [ExcludeFromCodeCoverage]
+    public class CosmosDbConnection
```

Comment:
You do not  need to define this interface yourself as it is defined in the Cosmos nuget